pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -742,6 +742,19 @@ func findNextInitContainerToRun(pod *v1.Pod, podStatus *kubecontainer.PodStatus)
 		return nil, nil, true
 	}
 
+	// If any of the main containers have status, then all init containers must
+	// after executed at some point in the past.  However, they could be removed
```

Comment:
Maybe "must have executed" instead of "must after executed"?

Reply:
ah yes, i'll fix it up before posting upstream